### PR TITLE
Fixes #11535 - add facet info to host commands

### PR DESCRIPTION
**Pull request by jlsherrill**
this adds content and subscription facet info
to the host info and host list commands

### CODE REVIEW

File: test/functional/host/extensions/info_test.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      par['id'] == '1'
+    end
+
+    json_file = File.join(File.dirname(__FILE__), 'data', 'host.json')
+    ex.returns(JSON.parse(File.read(json_file)))
+
+    param_ex = api_expects(:parameters, :index, 'Parameter index') do |par|
+      par[:host_id] == 1
+    end
+    param_ex.returns('results' => [])
```
Comment (akofink) [inline test/functional/host/extensions/info_test.rb]: There's an [`index_response` helper](https://github.com/theforeman/hammer-cli-foreman/blob/a7f86beb465737a4e8c4908a0fb9abfe824d0139/lib/hammer_cli_foreman/testing/api_expectations.rb#L53).

Comment (jlsherrill) [inline test/functional/host/extensions/info_test.rb]: sweet! updated

**Comment by akofink** [comment]
Works as expected. Nice tests. ACK

**Comment by jlsherrill** [comment]
Thanks for the review!

**Pull request merged**